src/main/java/hotel/Hotel.java

```diff
@@ -41,8 +42,9 @@ public void isCorrespond() {
                 System.out.println("Room corresponds to your criteria");
                 break;
             } else {
-                System.out.println("There is no rooms that matches your criteria");
+                throw new NoSuchElementException("There is no rooms that matches your criteria");
```

Comment:
NoSuchElementException or NoSuchRoomException ?

Reply:
fiixed to NoSuchRoom